### PR TITLE
[DD4hep] Replace TBB Containers with std Ones

### PR DESCRIPTION
**Pull request by ianna**
#### PR description:

As discussed during Sim meeting [here,](https://indico.cern.ch/event/962226/contributions/4047859/attachments/2120196/3568354/DD%20Core%20Migration%20Sim%20meeting%2009%20Oct%202020.pdf) current state of the XML files does not allow their parallel processing, so there is no need for TBB that has on overhead. This PR replaces them with std ones.

CMSSW_11_2_X_2020-10-09-1100:
```diff
TimeModule> 1 1 test DDCMSDetector 2.09666
TimeModule> 1 1 testVectors DDTestVectors 0.000801802
TimeModule> 1 1 testDump DDTestDumpFile 0.372976
TimeModule> 1 1 testGeoIter DDTestDumpGeometry 6.0661
TimeModule> 1 1 p PathStatusInserter 8.4877e-05
TimeModule> 1 1 TriggerResults TriggerResultInserter 3.29018e-05
TimeEvent> 1 1 8.53756
MemoryReport> Peak virtual size 672.719 Mbytes
rss  = 234.75 Mbytes
```
CMSSW_11_2_X_2020-10-09-1100 plus this PR:
```diff
TimeModule> 1 1 test DDCMSDetector 1.79474
TimeModule> 1 1 testVectors DDTestVectors 0.000830889
TimeModule> 1 1 testDump DDTestDumpFile 0.45456
TimeModule> 1 1 testGeoIter DDTestDumpGeometry 5.90832
TimeModule> 1 1 p PathStatusInserter 9.39369e-05
TimeModule> 1 1 TriggerResults TriggerResultInserter 3.29018e-05
TimeEvent> 1 1 8.15958
MemoryReport> Peak virtual size 708.672 Mbytes
rss = 226.383 Mbytes
```

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Before submitting your pull requests, make sure you followed this checklist:
- verify that the PR is really intended for the chosen branch
- verify that changes follow [CMS Naming, Coding, And Style Rules](http://cms-sw.github.io/cms_coding_rules.html)
- verify that the PR passes the basic test procedure suggested in the [CMSSW PR instructions](https://cms-sw.github.io/PRWorkflow.html)

### CODE REVIEW

**Comment by ianna** [comment]
please check

**Comment by ianna** [comment]
please test

File: DetectorDescription/DDCMS/interface/DDParsingContext.h [hunk 1]
```diff
@@ -83,9 +58,23 @@ namespace cms {
     bool debug_volumes = false;
     bool debug_placements = false;
     bool debug_namespaces = false;
-    bool debug_visattr = false;
     bool debug_algorithms = false;
     bool debug_specpars = false;
+
+    std::atomic<dd4hep::Detector*> description;
```
Comment (Dr15Jones) [inline DetectorDescription/DDCMS/interface/DDParsingContext.h:64]: Why still atomic?

Comment (ianna) [reply DetectorDescription/DDCMS/interface/DDParsingContext.h:64]: Thanks, @Dr15Jones !

**Comment by ianna** [comment]
please test

**Comment by ianna** [comment]
+1

**Comment by silviodonato** [comment]
+1

**Pull request merged**